cinema/views.py

```diff
@@ -2,15 +2,14 @@
 
 from django.db.models import F, Count
 from rest_framework import viewsets, mixins, status
-from rest_framework.authentication import TokenAuthentication
 from rest_framework.decorators import action
 from rest_framework.pagination import PageNumberPagination
-from rest_framework.permissions import IsAuthenticated, IsAdminUser
+from rest_framework.permissions import IsAdminUser
 from rest_framework.response import Response
-from rest_framework.viewsets import GenericViewSet, ReadOnlyModelViewSet
-
+from rest_framework.viewsets import GenericViewSet
+from rest_framework.viewsets import GenericViewSet
```

Comment:
The `GenericViewSet` is imported twice. You can remove the redundant import on line 10.